public/src/pages/auth/Login/Login.tsx

```diff
@@ -74,9 +76,16 @@ export const Login: FC = () => {
         setLoginResponse(data);
         return data;
       })
-      .then(({ email, errorText }) => {
-        setErrorText(errorText);
+      .then(async ({ email, errorText }) => {
```

Comment:
`async` statement is redudndat here

```suggestion
      .then(({ email, errorText }) => {
```